explorer/src/interfaces/swr/enum.ts

```diff
@@ -6,10 +6,11 @@ export enum SWRKeys {
   GET_RELATED_ATTESTATION = "getRelatedAttestations",
   GET_ATTESTATION_LIST = "getAttestationList",
   GET_ATTESTATION_COUNT = "getAttestationCount",
-  GET_RECENT_ATTESTATION = "getRecentAttestations",
+  GET_RECENT_ATTESTATION_SCHEMA = "getRecentAttestationsSchema",
+  GET_RECENT_ATTESTATION_PORTAL = "getRecentAttestationsPortal",
+  GET_RECENT_ATTESTATION_GLOBAL = "getRecentAttestations",
   GET_MODULE_LIST = "getModuleList",
   GET_MODULE_COUNT = "getModuleCount",
-  SEARCH = "search",
```

Comment:
This key should not be deleted (cf. build job failing)

Reply:
done